app/controllers/merchants_controller.rb

```diff
@@ -0,0 +1,56 @@
+class MerchantsController < ApplicationController
+  before_action :require_user, only: [:edit, :show]
+  before_action :redirect_if_logged_in, only: [:new, :create]
+
+  def index
```

Comment:
Rails magic opportunity! You don't have to define this method if you're not doing anything with it. Rails will automatically know what view to display if the method isn't defined & the view isn't unique.